src/components/InfoTip/InfoTip.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Dropdown } from '../Dropdown';
+
+class InfoTip extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleOnKeyDown = this.handleOnKeyDown.bind(this);
+    this.handleOnClick = this.handleOnClick.bind(this);
+    this.state = { open: false, footer_focused: false };
```

Comment:
Can you rename `footer_focused` to `footerFocused` please?

@priley86 Does the new linting rules going fix that?